### ISSUE TITLE
duplicate content in MANIFEST.MF

### ISSUE THREAD

**Issue by missedone** [cbeust/jcommander#435]
jcommander 1.73
all entries duplicate
<img width="1109" alt="jcommander" src="https://user-images.githubusercontent.com/555134/37134138-d6080dee-224b-11e8-9e4e-829f498ede87.png">

**Comment by jodastephen** [comment]
Just to note that `Automatic-Module-Name` didn't make it into the manifest either.
https://github.com/cbeust/jcommander/blob/master/build.gradle#L33

**Comment by missedone** [comment]
hi @cbeust , please fix the osgi header as well:
https://github.com/testng-team/testng-p2/pull/18#discussion_r154551752

**Comment by glhez** [comment]
The duplicate entry does not help Java either (removing the duplicate entry fix the problem): 

```
Exception in thread "main" java.lang.NoClassDefFoundError: com/beust/jcommander/IStringConverterFactory
        at java.lang.Class.getDeclaredMethods0(Native Method)
        at java.lang.Class.privateGetDeclaredMethods(Class.java:2701)
        at java.lang.Class.privateGetMethodRecursive(Class.java:3048)
        at java.lang.Class.getMethod0(Class.java:3018)
        at java.lang.Class.getMethod(Class.java:1784)
        at sun.launcher.LauncherHelper.validateMainClass(LauncherHelper.java:544)
        at sun.launcher.LauncherHelper.checkAndLoadMain(LauncherHelper.java:526)
Caused by: java.lang.ClassNotFoundException: com.beust.jcommander.IStringConverterFactory
        at java.net.URLClassLoader$1.run(URLClassLoader.java:370)
        at java.net.URLClassLoader$1.run(URLClassLoader.java:362)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.net.URLClassLoader.findClass(URLClassLoader.java:361)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:338)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 7 more
Caused by: java.io.IOException: invalid manifest format
        at java.util.jar.Manifest.read(Manifest.java:225)
        at java.util.jar.Manifest.<init>(Manifest.java:69)
        at java.util.jar.JarFile.getManifestFromReference(JarFile.java:194)
        at java.util.jar.JarFile.getManifest(JarFile.java:180)
        at sun.misc.URLClassPath$JarLoader$2.getManifest(URLClassPath.java:981)
        at java.net.URLClassLoader.defineClass(URLClassLoader.java:450)
        at java.net.URLClassLoader.access$100(URLClassLoader.java:73)
        at java.net.URLClassLoader$1.run(URLClassLoader.java:368)
        ... 13 more
```

Also, the jar content contains empty directories (kobaltBuild):

![image](https://user-images.githubusercontent.com/10855953/37313929-ac792954-2651-11e8-9078-58f671a2f4a4.png)

**Comment by glhez** [comment]
My error was referenced here: https://github.com/cbeust/jcommander/issues/434
(however this ticket was not mentioning it but it is also a duplicate of #434).

**Comment by talios** [comment]
I mentioned this on https://github.com/cbeust/jcommander/commit/85cb6f7217e15f62225185ffd97491f34b7b49bb#comments awhile ago when I spotted it.  I should have raised it as an actual issue tho - just came here to do so and found this ticket.

Be good to see a fix for this released.

**Comment by ketan** [comment]
I'm seeing this error as well (on 1.75). A lot of tooling that reads manifest files simply breaks when this manifest file is on the classpath.

**Comment by ebayedq** [comment]
> 
> 
> I'm seeing this error as well (on 1.75). A lot of tooling that reads manifest files simply breaks when this manifest file is on the classpath.

yup, this includes compiling via maven in intellij.

**Comment by cbeust** [comment]
Fixed in 1.78.
